src/screens/Music/stacks/Singles.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useCallback, useEffect } from 'react';
+import { useRoute, RouteProp } from '@react-navigation/native';
```

Comment:
This import (and a bunch of others in this file) is unused. Please remove any unused imports. Please check the output of `npm run lint` as well.